static/js/dist/app/settings.min.js

```diff
@@ -1 +1,233 @@
-$(document).ready(function(){$("#apiResetForm").submit(function(e){return api.reset().success(function(e){user.api_key=e.data,successFlash(e.message),$("#api_key").val(user.api_key)}).error(function(e){errorFlash(e.message)}),!1}),$("#settingsForm").submit(function(e){return $.post("/settings",$(this).serialize()).done(function(e){successFlash(e.message)}).fail(function(e){errorFlash(e.responseJSON.message)}),!1});var e=localStorage.getItem("gophish.use_map");$("#use_map").prop("checked",JSON.parse(e)),$("#use_map").on("change",function(){localStorage.setItem("gophish.use_map",JSON.stringify(this.checked))})});
\ No newline at end of file
+$(document).ready(function () {
```

Comment:
No need to remove it now if it's helping you test, but before we merge I'd ask that you please remove the *.min.js files from the PR. I'll rebuild everything on my end after merging the initial PR.

Reply:
Ah yes, I'm never sure how to handle `.min.js` files in my commits. Shall I delete the min files in my PR, or check them out from their originals?